src/geo_traits_impl.rs

```diff
@@ -0,0 +1,699 @@
+use crate::{
+    Multipoint, MultipointM, MultipointZ, Point, PointM, PointZ, PolygonRing, Polyline, PolylineM,
+    PolylineZ, NO_DATA,
+};
+use geo_traits::{
+    CoordTrait, LineStringTrait, MultiLineStringTrait, MultiPointTrait, MultiPolygonTrait,
+    PointTrait, PolygonTrait,
+};
+
+// Shapefile points can't be null, so we implement both traits on them
+impl CoordTrait for Point {
+    type T = f64;
+
+    fn dim(&self) -> geo_traits::Dimensions {
+        geo_traits::Dimensions::Xy
+    }
+
+    fn nth_unchecked(&self, n: usize) -> Self::T {
```

Comment:
feels like this could have been an unsafe method in the trait declaration :thinking: 
just like `interior_unchecked`

Reply:
Yes, that was fixed yesterday in https://github.com/georust/geo/pull/1242